stanford/stage-ssh-config

```diff
@@ -1,20 +1,8 @@
-#### edx-stage VPC 
-
-Host 54.241.183.3
-#Host vpc-jumpbox
-    HostName 54.241.183.3
-    User ubuntu
-    ForwardAgent yes
-
-Host *.us-west-1.compute.internal
-    User ubuntu
-    ForwardAgent yes
-    ProxyCommand ssh -W %h:%p ubuntu@54.241.183.3
-
 Host *
```

Comment:
FWIW: Since we're only using this to route through the jumpbox,
we can probably drop this file once we bump to Ansible2.0.

http://docs.ansible.com/ansible/faq.html#how-do-i-configure-a-jump-host-to-access-servers-that-i-have-no-direct-access-to